src/app/tests/TestCommandInteraction.cpp

```diff
@@ -475,242 +475,6 @@ void TestCommandInteraction::AddInvokeResponseData(nlTestSuite * apSuite, void *
     }
 }
 
-void TestCommandInteraction::TestCommandSenderWithWrongState(nlTestSuite * apSuite, void * apContext)
```

Comment:
There should be no ordering dependencies between tests here... If there are, that needs to be fixed.